policy/resolver_test.go

```diff
@@ -1451,3 +1453,54 @@ queries:
 		}
 	})
 }
+
+func TestResolve_TwoMrns(t *testing.T) {
+	b := parseBundle(t, `
+owner_mrn: //test.sth
+policies:
+- uid: policy1
+  groups:
+  - type: chapter
```

Comment:
Can we add one more check where the code id matches but the filter does not match the asset

Reply:
Added.